Octokit/Models/Response/RepositoryStar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents additional information about a star (such as creation time)
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryStar
+    {
+        public RepositoryStar() { }
```

Comment:
Ugh, that's a good point - I had a look at some other response models and found they also have an empty `ctor` - hold off on making that change, I'll let @Haacked refresh my memory here...